binplace.targets

```diff
@@ -35,6 +35,9 @@
       <TargetingPackDir Include="$(RefPath)" />
       <TargetingPackDir Condition="'$(IsNETCoreAppRef)' == 'true'" Include="$(BinDir)netcoreapp\pkg\ref" />
     </ItemGroup>
+    <ItemGroup Condition="'$(NuGetTargetMoniker)' == '' Or $(NuGetTargetMoniker.Contains('UAP'))">
```

Comment:
No this one should be based on BuildConfiguration, as we want to consistently create the targeting pack folders per BuildConfiguration.